### PR TITLE
Js ipfs0.30

### PR DESCRIPTION
**Pull request by daviddias**
Depends on:
- [x] https://github.com/ipfs-shipyard/ipfs-pubsub-room/pull/53

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,17 +7,23 @@
...
   "homepage": "https://github.com/pgte/y-ipfs-connector#readme",
   "dependencies": {
-    "async": "^2.4.1",
-    "debug": "^2.6.7",
-    "ipfs-pubsub-room": "^0.1.4",
-    "libp2p-crypto": "^0.8.8",
-    "safe-buffer": "^5.1.1"
+    "async": "^2.6.1",
+    "debug": "^3.1.0",
+    "ipfs-pubsub-room": "ipfs-shipyard/ipfs-pubsub-room#js-ipfs0.30",
```
Comment (daviddias) [inline package.json]: - [ ] Change this before merge

**Comment by pgte** [comment]
@diasdavid released version 1.4.0 of ipfs-pubsub-room.

**Comment by daviddias** [comment]
@pgte ready for that review, merge and publish

**Review by pgte — APPROVED** [review]
LGTM

**Comment by pgte** [comment]
Landed on v2.3.0

**Comment by daviddias** [comment]
don 't forget to push to master

**Pull request merged**